### ISSUE TITLE
Lightbox & Docker

### ISSUE THREAD

**Issue by Beee4life** [archetyped/simple-lightbox#744]
I'm having trouble running this plugin in a Docker environment/container.

I noticed that when I use a Docker container, the lightbox won't show. I see all the lightbox code and all needed files are enqueued.

I have tested this in 3 environments, of course with the same repository/code;
- local with Docker (not working)
- local without Docker (working)
- live (working)

I don't see any errors in my console, also reverting to a default WP template and no plugins didn't do the trick.

Does my server config maybe 'miss' a setting/package ?

**Comment by archetyped** [comment]
Hi, without being able to test it myself, I can only guess that it may have to do with the Docker container's cURL configuration.  See [SLB's Requirements documentation](https://github.com/archetyped/simple-lightbox/wiki/Requirements#curl-support) for more details on how cURL needs to be configured for WordPress (SLB uses WordPress' built-in functionality).

If you create a simple page (e.g. with a single image link) and post the page's source, I would be glad to take a closer look to see if there is anything else that can be determined.

**Comment by Beee4life** [comment]
I thought I had replied already but apparently I hadn't.

I deduced it had to do with cURL, because it still works on a live environment but not on dev. What I don't get is why does a 'simple lightbox plugin' needs to cURL (to) itself ?

WPML is throwing similar issues in my face (but doesn't stop working). I also experienced (unsolved) issues with creating a custom lightbox design for this plugin, so I'm sorry to say it but I have dropped this plugin all together and phasing it out in favor of lightbox.js by Lokesh Dhakar.

Much lighter... Thanks for the plugin for quite some years, but I have found an alternative.
